### PR TITLE
Update changelog for v2.7.0-rc1

### PR DESCRIPTION
**Pull request by nubeiro**
This pull request should be the last merge for v2.7.0-rc1. 
It documents in CHANGELOG.md file notable changes since last release.

### CODE REVIEW

**Comment by nubeiro** [comment]
@michaelklishin if you could take a look before I merge and tag I'd appreciate it ;-)

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ All Notable changes to `php-amqplib` will be documented in this file
 
 ## [Unreleased]
 
+## 2.7.0-rc1 - 2017-05-22
+### Added
+- Increased overall test coverage.
+- Bring heartbeat support to socket connection
+- Add message delivery tag for internal ack handler
```
Comment (michaelklishin) [inline CHANGELOG.md]: It would make more sense to the users if we changed it to say "for publisher confirms".

**Comment by nubeiro** [comment]
Thank you @michaelklishin 
I've just created release for v2.7.0-rc1

**Comment by HonzaMac** [comment]
When will be stable `v2.7.0` ?

**Pull request merged**